engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -186,4 +209,30 @@
 	 * @since 5.2
 	 */
 	HibernateValidatorConfiguration timeProvider(TimeProvider timeProvider);
+
+	/**
+	 * Allow to set a {@code MethodValidationConfiguration} to modify the behavior of the Validator with respect to
+	 * Specification section 4.5.5. In particular:
+	 * <pre>
+	 * "Out of the box, a conforming Bean Validation provider must throw a
+	 * ConstraintDeclarationException when discovering that any of these rules are violated.
+	 * In addition providers may implement alternative, potentially more liberal, approaches
+	 * for handling constrained methods in inheritance hierarchies. Possible means for activating
+	 * such alternative behavior include provider-specific configuration properties or annotations.
+	 * Note that client code relying on such alternative behavior is not portable between Bean
+	 * Validation providers."
+	 * </pre>
+	 *
+	 * @param mvc {@code MethodValidationConfiguration} instance with the method validation behaviour configuration
+	 * settings.
+	 *
+	 * @return {@code this} following the chaining method pattern
+	 */
+	HibernateValidatorConfiguration setMethodValidationConfiguration(MethodValidationConfiguration mvc);
```

Comment:
> The proposed design seems to break the fluent invocation style

Right. The original pull request has separate setters as well, but then the values which got internally passed around were blowing up the method signatures. For this reason I suggested to introduce `MethodValidationConfiguration` as a parameter object, but I guess this does not have to be exposed externally to the `Configuration`, but can be created internally. I'll update the pull request.